test/build_and_test.sh

```diff
@@ -12,12 +12,18 @@ source $(dirname $0)/test_utils.sh
 
 IMAGE=gcr.io/fission-ci/fission-bundle
 FETCHER_IMAGE=gcr.io/fission-ci/fetcher
+PYTHON_RUNTIME_IMAGE=gcr.io/fission-ci/python-env
```

Comment:
We should generalize this a bit, accepting 'python' as a parameter instead of hard-coding it. That way we can extend these tests to other languages more easily.